addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss

```diff
@@ -147,6 +152,20 @@
         background-color: rgba(0, 0, 0, 0.8);
         color: lighten(gray('400'), 15%);
     }
+    &.btn-primary {
+        @include button-variant($o-brand-primary, $o-brand-primary);
+    }
+    @include media-breakpoint-down(sm) {
+        &.btn {
+            padding: 0;
+            border: 0;
+            background: none;
+
+            span {
+                display: none;
+            }
+        }
+    }
```

Comment:
I suppose this it to target mobile? Please make use of `o-mobile` that we have (or introduce it if it's missing on this component).